File: Codeforces/Kotlin/src/main/kotlin/problems/UltraFastMathematician.kt
package problems.problems

/* Problem Link: https://codeforces.com/problemset/problem/61/A */

fun main() {
    val first = readln()
    val second = readln()
    first.forEachIndexed { index, c ->
        print(if (c == second[index]) '0' else '1')
    }
}

File: Codeforces/C++/problemset/Ultra-Fast Mathematician.cpp
// Problem Link: https://codeforces.com/problemset/problem/61/A

#include <iostream>
#include <string>
using namespace std;
int main()
{
    string x,y;
    cin >> x >> y;
    for(int i = 0; i < x.size(); i++)
    {
        if(x[i] == y[i])
            cout << 0;
        else
            cout << 1;
    }


    return 0;
}
